dubbo-dependencies-bom/pom.xml

```diff
@@ -116,7 +116,7 @@
         <jetty_version>9.4.43.v20210629</jetty_version>
         <validation_new_version>3.0.1</validation_new_version>
         <validation_version>1.1.0.Final</validation_version>
-        <hibernate_validator_version>5.4.1.Final</hibernate_validator_version>
```

Comment:
I am more worried about the incompatibility between 5.x and 6.x. Is it reasonable to upgrade to 5.4.2?

Reply:
I'm not sure why dependabot suggested upgrading to hibernate-validator 6 - you're right that using a new v5 release is enough to fix the vulnerability. There is a v5.4.3 - I've upgraded to that, if that's ok.